src/classes/frDonorEmails.cls

```diff
@@ -0,0 +1,63 @@
+public class frDonorEmails extends frModel {
+    public static final String TYPE = 'DonorEmails';
+
+    public static List<frMapping__c> mappings {
```

Comment:
I didn't see a way to create these mappings at all with the package installed. Are we intending to let users customize how data about the email flows into salesforce or will we just be populating things with our own defaults?

Reply:
I copied it over because it was needed to extend frModel, but I actually don't need frModel.  I am removing all overrides from frModel.